ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -116,6 +116,7 @@ Item {
                     nameInput.input.edit.clear();
                 }
```

Comment:
Could you please add above `Layout.preferredHeight: 78` ? this will prevent the text input of moving when there is an error message to be shown :)

Reply:
@alexandraB99 yes, sure. Updated.